src/calendar-list/index.tsx

```diff
@@ -216,13 +216,16 @@ const CalendarList = (props: CalendarListProps, ref: any) => {
     return false;
   }, [currentMonth]);
 
-  const renderItem = useCallback(({item}: any) => {
+  const renderItem = useCallback(({item}: {item: XDate}) => {
+    const dateString = toMarkingFormat(item);
+    const [year, month] = dateString.split('-');
```

Comment:
If `item` is of XDate type you don't need to move it to be a string and split it, you can just use its methods, like `item.getFullYear()` and `item.getMonth()`.

Reply:
I did it at first, but the `getMonth` returns a zero based month number so I had to add +1 and it also without the zero padding..
So at the end it was just cleaner to use markingFormat

Reply:
Is it possible this is the reason you get testID with `item_**month-year**.day_**date**` that don't match?

Reply:
What do you don't match? 
All the tests pass. 

Not sure I follow

Reply:
I meant this: expandableCalendar.calendarList.item_**2022-08**.day_**2022-09**-01 - the day is in September and the item is in August